apps/web/src/features/targetedOutreach/constants.ts

```diff
@@ -1,5 +1,12 @@
+import { FEATURES } from '@/utils/chains'
+
 export const ACTIVE_OUTREACH = { id: 2, url: 'https://wn2n6ocviur.typeform.com/to/nlQlP7lU', targetAll: true }
 
+export const TARGETED_FEATURES = [
+  { id: 3, feature: FEATURES.NESTED_SAFES },
+  { id: 4, feature: FEATURES.MASS_PAYOUTS },
+] as const
```

Comment:
It's a little bit confusing that the outreach popup feature is mixed with a more generic "targeted features" feature.

The idea of [feature folders](https://www.notion.so/safe-global/Feature-toggles-970c96d18e544fd1a01a362f0535146b) was to isolate features so that they can be added, removed and toggled independently from everything else. With this change, the outreach popup will be coupled with targeting.

P.S. btw "a targeted Safe" is an ominous combination of words.

Reply:
Everything has been renamed/relocated in 927d53384247b970800270b9d19992306ea79470.

- `useIsTargetedSafe` -> `useIsOutreachSafe`
- Feature-targeting logic -> `@/features/targetedFeatures`

Reply:
Thanks! However, you're still importing and using a hook from the popup feature which creates a strange dependency. That they use the same endpoint doesn't mean we should be using a hook to check whether to render a survey popup to toggle other features.